security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -15,6 +15,24 @@ For an overview of the File Integrity Operator, see xref:../../security/file_int
 
 To access the latest release, see xref:../../security/file_integrity_operator/file-integrity-operator-updating.adoc#olm-preparing-upgrade_file-integrity-operator-updating[Updating the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-1-3-6"]
+== OpenShift File Integrity Operator 1.3.6
+ 
+The following advisory is available for the OpenShift File Integrity Operator 1.3.6:
+
+* link:https://access.redhat.com/errata/RH??-2025:????[RH??-2025:???? OpenShift File Integrity Operator Bug Fix Update]
+
+[id="file-integrity-operator-1-3-6-bug-fixes"]
+=== Bug fixes
+
+* Previously, FIO would re-initialize on failure for all nodes in the cluster. With this release, FIO has been updated to re-initialize the database on failure nodes only.  (link:https://issues.redhat.com/browse/OCPBUGS-18933[*OCPBUGS-18933*])
+
+* Previously, after a reset of FIO, the metric file_integrity_operator_node_failed would be reset, which would cause an existing alert `NodeHasIntegrityFailure` to disappear. With this release, `NodeHasIntegrityFailure` is preserved after an FIO restart. (link:https://issues.redhat.com/browse/OCPBUGS-42807[*OCPBUGS-42807*])
+
+* Previously, when a new node is added to a cluster by scaling up the `machineset`, FIO would mark the newly added node as `Failed`. With this release FIO marks the newly added node as `Succeeded`. (link:https://issues.redhat.com/browse/OCPBUGS-36483[*OCPBUGS-36483*])
```

Comment:
In some cases, the newly added node might still fail an AIDE scan, but the fix was to wait to start AIDE until the node was ready (e.g., it was bootstrapped). We could say that making the startup process more defensive should reduce the likelihood of scan that trip immediately on files from the installation process.